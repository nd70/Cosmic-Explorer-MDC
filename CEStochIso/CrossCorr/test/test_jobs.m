stochastic_paths;
delete('test_jobs.out');
diary('test_jobs.out');
stochastic('old/input/paramfiles/test_params.txt', 'old/input/jobfiles/test_jobs.txt', 1);
diary off;
