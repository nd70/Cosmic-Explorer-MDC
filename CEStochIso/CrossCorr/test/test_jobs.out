segment=3, trial =1, start sec = 793177086, CC stat = -9.042982e-03, theor sigma = 2.988294e-02
segment=4, trial =1, start sec = 793177116, CC stat = -1.005975e-02, theor sigma = 3.020240e-02
segment=5, trial =1, start sec = 793177146, CC stat = 3.887399e-02, theor sigma = 2.759917e-02
segment=6, trial =1, start sec = 793177176, CC stat = 2.509199e-02, theor sigma = 2.780385e-02
segment=7, trial =1, start sec = 793177206, CC stat = -3.019547e-03, theor sigma = 2.763522e-02
segment=8, trial =1, start sec = 793177236, CC stat = 2.620551e-02, theor sigma = 2.787284e-02
segment=9, trial =1, start sec = 793177266, CC stat = 1.955863e-02, theor sigma = 2.897976e-02
segment=10, trial =1, start sec = 793177296, CC stat = -3.943661e-02, theor sigma = 2.961600e-02
segment=11, trial =1, start sec = 793177326, CC stat = -4.358672e-03, theor sigma = 2.868751e-02
segment=12, trial =1, start sec = 793177356, CC stat = 4.016484e-03, theor sigma = 2.770450e-02
segment=13, trial =1, start sec = 793177386, CC stat = 4.366201e-03, theor sigma = 2.884221e-02
segment=14, trial =1, start sec = 793177416, CC stat = 1.457833e-03, theor sigma = 2.920267e-02
segment=15, trial =1, start sec = 793177446, CC stat = 4.396108e-02, theor sigma = 2.782415e-02
segment=16, trial =1, start sec = 793177476, CC stat = -2.378528e-02, theor sigma = 2.673180e-02
segment=17, trial =1, start sec = 793177506, CC stat = 3.282716e-02, theor sigma = 2.658529e-02
segment=18, trial =1, start sec = 793177536, CC stat = -3.689884e-03, theor sigma = 2.654770e-02
segment=19, trial =1, start sec = 793177566, CC stat = 2.472387e-02, theor sigma = 2.626735e-02
segment=20, trial =1, start sec = 793177596, CC stat = -8.015115e-04, theor sigma = 2.585116e-02
segment=21, trial =1, start sec = 793177626, CC stat = 2.668306e-02, theor sigma = 2.608204e-02
segment=22, trial =1, start sec = 793177656, CC stat = -4.170815e-03, theor sigma = 2.598650e-02
segment=23, trial =1, start sec = 793177686, CC stat = 2.759887e-02, theor sigma = 2.539612e-02
segment=24, trial =1, start sec = 793177716, CC stat = 2.025520e-02, theor sigma = 2.547889e-02
segment=25, trial =1, start sec = 793177746, CC stat = 3.783273e-03, theor sigma = 2.625349e-02
segment=26, trial =1, start sec = 793177776, CC stat = 3.477117e-02, theor sigma = 2.660288e-02
segment=27, trial =1, start sec = 793177806, CC stat = -1.028810e-02, theor sigma = 2.617460e-02
segment=28, trial =1, start sec = 793177836, CC stat = 1.581485e-02, theor sigma = 5.230732e-02
segment=29, trial =1, start sec = 793177866, CC stat = 2.435789e-02, theor sigma = 5.218478e-02
segment=30, trial =1, start sec = 793177896, CC stat = 2.372809e-02, theor sigma = 2.685106e-02
segment=31, trial =1, start sec = 793177926, CC stat = -4.575046e-02, theor sigma = 2.733702e-02
